src/parser.ts

```diff
@@ -607,7 +622,7 @@ export class Parser {
           ))
       ) {
         let value = propType.types.map(type => ({
-          value: this.getValuesFromUnionType(type)
+          ...this.getInfoFromUnionType(type)
```

Comment:
Does it make sense to expand properties to the object? Why don't return just the object itself?
`let value = propType.types.map(type => this.getInfoFromUnionType(type))`

Reply:
> Does it make sense to expand properties to the object? Why don't return just the object itself?
> `let value = propType.types.map(type => this.getInfoFromUnionType(type))`

thanks for reply, 
sounds greater and  you are right, but it causes a breaking change~~~,so I haven't make a change for it.

Reply:
Sorry, I cannot see that breaking change in my proposal. You are exposing all properties to the new object. I am proposing just to skip that and return that object instead. Am I missing something?

Reply:
oh, I'm sorry I made a misunderstanding about your reply. the Destructuring assignment of this object are meaningless and unnecessary. Thank you for your patience to this PR~~~😀